### PR TITLE
detach from Figma's tokens

### PR DESCRIPTION
**Pull request by six7**
This is a WIP PR to detach from Figma's tokens and instead ship our own.

/describe

### CODE REVIEW

File: src/app/components/Button/StyledButton.tsx [hunk 1]
```diff
@@ -27,28 +27,25 @@ export const StyledButton = styled('button', {
...
+        borderColor: '$borderDefault',
         '&:focus-visible': {
           boxShadow: '$focus',
         },
       },
       ghost: {
         background: 'transparent',
         padding: '$2 $3',
-        color: '$fgBtnGhost',
-        '&:hover': {
```
Comment (LukeFinch) [inline src/app/components/Button/StyledButton.tsx:41]: Do ghost buttons not have hover states?

File: src/app/components/ThemeSelector/ThemeSelector.tsx [hunk 2]
```diff
@@ -110,7 +110,7 @@ export const ThemeSelector: React.FC = () => {
           return (
             filteredThemes.length > 0 && (
               <DropdownMenuRadioGroup value={typeof activeTheme[groupName] !== 'undefined' ? activeTheme[groupName] : ''}>
-                <Text css={{ color: '$staticTextMuted', padding: '$2 $3' }}>{groupName === INTERNAL_THEMES_NO_GROUP ? INTERNAL_THEMES_NO_GROUP_LABEL : groupName}</Text>
+                <Text css={{ color: '$contextMenuForegroundMuted', padding: '$2 $3' }}>{groupName === INTERNAL_THEMES_NO_GROUP ? INTERNAL_THEMES_NO_GROUP_LABEL : groupName}</Text>
```
Comment (LukeFinch) [inline src/app/components/ThemeSelector/ThemeSelector.tsx:113]: This token name doesn't match the naming convention of 'fg' for foreground

File: src/hooks/useFigmaTheme.ts [hunk 3]
```diff
@@ -5,8 +5,21 @@ export function useFigmaTheme() {
 
   useEffect(() => {
     const htmlClassList = document.documentElement?.classList || [];
+    console.log('useFigmaTheme', isDarkTheme, htmlClassList);
```
Comment (LukeFinch) [inline src/hooks/useFigmaTheme.ts:8]: Do we need the console log here in production?

**Review by LukeFinch — APPROVED** [review]
Only a couple of minor comments - nothing major.
This looks really good - and will be super helpful for iterating designs!

File: src/app/components/ThemeSelector/ThemeSelector.tsx (hunk 2 above)
Comment (six7) [reply src/app/components/ThemeSelector/ThemeSelector.tsx:113]: 💯  i will create a followup PR for a second refactor just to keep this PR a bit smaller. i'd also like to change `textMuted / textDefault` over to `fgDefault` etc

**Pull request merged**